### ISSUE TITLE
Trainer - Professional Summary - Text Area showing wrong UI

### ISSUE THREAD

**Issue by saknid** [Arunideas/React-app#15]
When user visits Professional Summary page, it should show text area with text editor. Once, user enters any data clicks out side it should show entered text as paragraph (in other term save text on ui) and hide the editor. 

**Testcase Used - Trainer Profile -- Test Case-10/Steps - 7**

Based upon this below are the issues - 

- For now it showing text editor as separate function. 
- It seems to be a mock design as text editor is not functional and entered values can not be seen as saved data above editor.

**Screen Shot Attached.**

![Image](https://user-images.githubusercontent.com/39871181/216806379-4850103f-8611-46ee-a6f3-b02906bb87e9.jpg)

**Comment by vishal1145** [comment]
It has been replace with syncfusion editor

**Comment by saknid** [comment]
@Arunideas Needed discussion for this item.

**Comment by saknid** [comment]
@Arunideas One small question do we keep text area open with editor even after save action is performed?

**Comment by Arunideas** [comment]
yeah - It should be open as is. I want to see one example that shows
different text formatings including showing the link , bullets, alignment,
bold italic , colored text , etc.

<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
Virus-free.www.avast.com
<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
<#DAB4FAD8-2DD7-40BB-A1B8-4E2AA1F9FDF2>
